slackclient/server.py

```diff
@@ -254,8 +256,11 @@ def api_call(self, method, timeout=None, **kwargs):
                 u'{"ok":false,"error":"channel_not_found"}'
 
             See here for more information on responses: https://api.slack.com/web
-        '''
-        return self.api_requester.do(self.token, method, kwargs, timeout=timeout).text
```

Comment:
aw crap. on closer inspection, it looks like this change is a breaking one, right? the method used to return a simple string, and now it returns a dict. is this going to work?

Reply:
oh wait, it still returns a string, but its the serialized JSON? that doesn't seem very friendly, because that means just to get the same functionality i used to get i have to parse the JSON string and lookup the `text` property myself. still a breaking a change.

Reply:
i always thought this method returned a dict, now i sad 😢

Reply:
It grabs `.text`, parses the JSON, appends to it, then dumps it to a string and returns. The payload's the same text + header

Reply:
AH AM I JUST AN IDIOT? did it always return a string which needed to be JSON parsed? if so, then we are okay!

Reply:
okay, plz ignore my half-baked thoughts.